core/src/main/java/org/apache/calcite/rel/core/Correlate.java

```diff
@@ -108,7 +108,7 @@ public Correlate(RelInput input) {
     this(
         input.getCluster(), input.getTraitSet(), input.getInputs().get(0),
         input.getInputs().get(1),
-        new CorrelationId((Integer) input.get("correlationId")),
+        new CorrelationId((Integer) input.get("correlation")),
```

Comment:
Should we keep `correlationId` and serialize `correlationId` instead of `correlation`?

Reply:
Yes, that's better.
I update the PR.
Thanks for reviewing.